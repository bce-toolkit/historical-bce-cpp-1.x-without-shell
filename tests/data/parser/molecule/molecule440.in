Na4XeO6.2H2O
