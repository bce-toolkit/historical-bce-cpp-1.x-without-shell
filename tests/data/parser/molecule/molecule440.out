H 4
Na 4
O 8
Xe 1
